input/resources/valueset-rsg-type.xml

```diff
@@ -12,11 +12,11 @@
         <system value="urn:ietf:rfc:3986"/>
         <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.31"/>
     </identifier>
-    <name value="CommonAURecordedSexOrGenderType" />
-    <title value="Common AU Recorded Sex or Gender Type" />
+    <name value="AURecordedSexOrGenderType" />
+    <title value="AU Recorded Sex or Gender Type" />
     <status value="draft" />
     <experimental value="false"/>
-    <description value="The Common AU Recorded Sex or Gender Type value set includes codes that can be used to represent the type or category of sex or gender that is recorded." />
+    <description value="The AU Recorded Sex or Gender Type value set includes codes that can be used to represent the type or category of sex or gender that is recorded." />
```

Comment:
This change is not related to FHIR-46107, please revert.

Reply:
Reverted change

Reply:
Change reverted